packages/commons/src/sanitization/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as express from "express";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable functional/immutable-data */
```

Comment:
Are these needed? If yes, let's keep them confined near the line where they are needed using `eslint-disable-next-line`

```suggestion
```